mitodl/ocw-studio
# Added minimal markdown as site content widget option

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes #133

#### What's this PR do?
Adds the minimal markdown editor (#112) as a widget option

#### How should this be manually tested?
Define a site config for your website starter and specify `markdown_minimal` as the `widget` value for a field. If you go to the editing screen for that content type, you should see the minimal markdown editor, and saving the form should work as expected

#### Screenshots (if appropriate)
![ss 2021-03-22 at 18 07 09 ](https://user-images.githubusercontent.com/14932219/112064836-fd0fcd00-8b39-11eb-9026-843471c14a37.png)
